website/blog/2020-05-18-rn4mupdadates.md

```diff
@@ -11,9 +11,9 @@ We're on React Native core version 0.62 with new community modules, native modul
 
 <!--truncate-->
 
-## Upgraded to 0.62
+## Version 0.62-preview Released

```

Comment:
Remove the word "Released". We can announce the release of stable version in a second blog later.

Reply:
Removed.